panel/dist/css/button.css

```diff
@@ -136,3 +136,10 @@
   background-color: var(--light-bg-color);
   color: black;
 }
+
+.bk-btn a {
```

Comment:
You also need to remove the padding from the button. I.e. set it to 0px.

I think you need to make this more specific targetting the specifically FileDownload?

Reply:
@philippjfr

Reply:
Wouldn't want to delete the padding from the general Button that is true, the `.bk-btn a` rule is fine because a regular Button doesn't support links inside it.

Reply:
Is the padding on `a` needed at all?

Reply:
It was for me. It aligns the text center vertically. That is also why it was on the button in the first place.